frontend/ui/detailgraph.jsx

```diff
@@ -1,12 +1,18 @@
-import _ from 'lodash';
 import React from 'react';
+import _ from 'lodash';
 import Dimensions from 'react-dimensions';
 import { curveLinear } from 'd3';
 import { timeFormat } from 'd3-time-format';
 import MetricsGraphics from 'react-metrics-graphics';
 
 
-class MeasureGraph extends React.Component {
+class DetailGraph extends React.Component {
+  shouldComponentUpdate(nextProps) {
+    return !_.isEqual(this.props.seriesList, nextProps.seriesList) ||
+      this.props.containerWidth !== nextProps.containerWidth ||
+      this.props.containerHeight !== nextProps.containerHeight;
```

Comment:
I have not been able to notice a difference between production and this branch when it comes to resizing the window size.
Maybe it is happening. I just wanted to let you know in case you wanted to know.

Reply:
Well production was working fine because it would resize/redraw the graph on all state changes, including dimension ones. :) This way we only resize if there is a tangible change in layout of the page.